test/home.test.js

```diff
@@ -2,15 +2,14 @@ import chai from 'chai';
 import chaiHttp from 'chai-http';
 
 import server from '../src/index';
-import env from '../src/configs/environments';
 
```

Comment:
@Ramadhan0 Not sure why you have changes in this file.

Reply:
This file was testing Dotenv, that file contain Dotenv configurations.